### PR TITLE
step 2.1.0-next -> 2.2.0

### PR DESCRIPTION
**Pull request by AlvaroVega**
Mimics https://github.com/telefonicaid/iotagent-manager/pull/309

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +0,0 @@
-- Remove: RPM stuff (no longer used)
-- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iotagent-node-lib 3.2.0) (iota-node-lib#1384)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:2]: To be moved to rpm/SPECS/iotamanager.spec, along with the entry about the upgrade in the library version.

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:2]: rpm/SPECS/iotamanager.spec was removed by https://github.com/telefonicaid/iotagent-manager/pull/311

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:2]: Use Changelog file for this

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:2]: fixed in https://github.com/telefonicaid/iotagent-manager/pull/313/commits/88bc7019d21a689b7b4f8137eb9d5226eba4b799

File: Changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+2.2.0 (Aug 24th, 2023)
+
+- Remove: RPM stuff (no longer used)
+- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, 
+  after the removal of legacy expressions in iotagent-node-lib 3.2.0) (iota-node-lib#1384)
```
Comment (fgalan) [inline Changelog:5]:
```suggestion
2.2.0 (Aug 25th, 2023)

- Upgrade iotagent-node-lib dependency from 3.2.0 to 3.3.0
- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iotagent-node-lib 3.2.0) (iota-node-lib#1384)
- Remove: RPM stuff (no longer used)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**